.travis.yml

```diff
@@ -5,6 +5,9 @@ notifications:
     on_success: never
     on_failure: change
 
+env:
```

Comment:
I'm not familiar enough with the Atom toolchain to know this, so I'll ask: what's the significance of adding this? Is it necessary to fix this issue?